andela/blade-runner-cfh
# #153741756 Users should receive a JWT on successful signup

#### What does this PR do?
>- This PR ensures a user receives a JSON web token on successful sign-up
#### Description of Task to be completed?
>- Users once authenticated via the signup API should be returned a JWT to be required on subsequent calls to all other endpoints that require authentication.

#### How should this be manually tested?
>- Open postman, input the URL address which is "localhost:3000/api/users"
>- input the following parameters:
    name: 'string'
    username: 'string'
    email: 'string'
    password: 'string'
    avatar: 'string'

>- send the request to the server
>- you should get a response containing the JSON web token

#### Any background context you want to provide?
>- N/A

#### What are the relevant pivotal tracker stories?
>- https://www.pivotaltracker.com/n/projects/2137297/stories/153741756

#### Screenshots (if appropriate):
<img width="1125" alt="screen shot 2018-01-04 at 11 19 12 pm" src="https://user-images.githubusercontent.com/22421283/34586818-dbd0e01a-f1a5-11e7-8213-cfb8a68e7300.png">


#### Questions:
>- N/A
